app/file-upload/server/lib/FileUpload.js

```diff
@@ -265,6 +265,41 @@ export const FileUpload = {
 		return result;
 	},
 
+	createImageThumbnail(file) {
+		if (!settings.get('Message_Attachments_Thumbnails_Enabled')) {
+			return;
+		}
+
+		file = Uploads.findOneById(file._id);
+		file = FileUpload.addExtensionTo(file);
+		const store = FileUpload.getStore('Uploads');
+		const image = store._store.getReadStream(file._id, file);
+		const width = settings.get('Message_Attachments_Thumbnails_Width');
+		const height = settings.get('Message_Attachments_Thumbnails_Height');
+
+		const transformer = sharp()
+			.resize({ width, height, fit: 'inside' })
+			.webp();
```

Comment:
@diegolmello is WebP fine for mobile?

Reply:
No, iOS doesn't provide support for it.
I've read that they'd start supporting it on iOS 14, but that's the current version so we'd be dropping support to recent older versions.

Reply:
ok thx.. so @KevLehman I think we'll need to change this to keep the original format, ok?

Reply:
Nice, I'll do the change!

Reply:
Done!